Taskfile.yml

```diff
@@ -372,57 +373,72 @@ tasks:
       # excluding the ./apis directory here
```

Comment:
I'm not sure this is true anymore, it's not excluded. I don't know why it should be excluded though because I do think that the controller does depend on the api folder... so maybe just delete this comment?